app/javascript/dashboard/i18n/locale/en/report.json

```diff
@@ -59,6 +59,29 @@
     "CUSTOM_DATE_RANGE": {
       "CONFIRM": "Apply",
       "PLACEHOLDER": "Select date range"
+    },
+    "GROUP_BY_FILTER_DROPDOWN_LABEL": "Group By",
+    "GROUP_BY_DAY": "day",
+    "GROUP_BY_WEEK": "week",
+    "GROUP_BY_MONTH": "month",
+    "GROUP_BY_YEAR": "year",
+    "GROUP_BY_OPTIONS": {
```

Comment:
We need to move this object to a separate constants file and use use locale files for labels and text. Since we are using this object in `app/javascript/dashboard/routes/dashboard/settings/reports/Index.vue:165`, it might break in later if other locale files are not matching.

Reply:
@pranavrajs can we take this as a separate task?